pkg/providers/cloudstack/cloudstack.go

```diff
@@ -36,6 +36,7 @@ import (
 const (
 	eksaLicense                = "EKSA_LICENSE"
 	controlEndpointDefaultPort = "6443"
+	cloudstackAnnotationSuffix = "cloudstack.anywhere.eks.amazonaws.com/v1alpha1"
```

Comment:
should we just make this public? Looks like it's defined in a couple places

Reply:
I find a better place to place this const, pkg/constants/constants.go